assets/controls/flatDarkB.png.import

```diff
@@ -0,0 +1,34 @@
+[remap]
```

Comment:
Strictly speaking the B, Up and Down assets in each colour scheme are not needed (but to me it seems harmless to have them. You never know what new features might need them later).

Reply:
Justin mentioned wanted having an alternate option there as part of the modability, so I think it can make sense to keep (similar to how we have multiple players sprites).

Reply:
I pushed a patch to use the inverted assets while the buttons are pressed.